monitoring/ceph-mixin/prometheus_alerts.libsonnet

```diff
@@ -0,0 +1,695 @@
+            |||,
+          },
+        },
+        {
+          alert: 'CephMonDiskspaceCritical',
+          'for': '1m',
+          expr: 'ceph_health_detail{name="MON_DISK_CRIT"} == 1',
+          labels: { severity: 'critical', type: 'ceph_default', oid: '1.3.6.1.4.1.50495.1.2.1.3.2' },
+          annotations: {
+            documentation: 'https://docs.ceph.com/en/latest/rados/operations/health-checks#mon-disk-crit',
+            summary: 'Disk space on at least one monitor is critically low%(cluster)s' % $.MultiClusterSummary(),
+            description: "The free space available to a monitor's store is critically low (<5% by default). You should increase the space available to the monitor(s). The default location for the store sits under /var/lib/ceph. Your monitor hosts are; {{- range query \"ceph_mon_metadata\"}}   - {{ .Labels.hostname }} {{- end }} ",
+          },
+        },
+        {
+          alert: 'CephMonDiskspaceLow',
+          'for': '5m',
+          expr: 'ceph_health_detail{name="MON_DISK_LOW"} == 1',
+          labels: { severity: 'warning', type: 'ceph_default' },
+          annotations: {
+            documentation: 'https://docs.ceph.com/en/latest/rados/operations/health-checks#mon-disk-low',
+            summary: 'Disk space on at least one monitor is approaching full%(cluster)s' % $.MultiClusterSummary(),
+            description: "The space available to a monitor's store is approaching full (>70% is the default). You should increase the space available to the monitor store. The default location for the store sits under /var/lib/ceph. Your monitor hosts are; {{- range query \"ceph_mon_metadata\"}}   - {{ .Labels.hostname }} {{- end }} ",
+          },
+        },
+        {
+          alert: 'CephMonClockSkew',
+          'for': '1m',
+          expr: 'ceph_health_detail{name="MON_CLOCK_SKEW"} == 1',
+          labels: { severity: 'warning', type: 'ceph_default' },
+          annotations: {
+            documentation: 'https://docs.ceph.com/en/latest/rados/operations/health-checks#mon-clock-skew',
+            summary: 'Clock skew across the Monitor hosts%(cluster)s detected' % $.MultiClusterSummary(),
+            description: "The ceph monitors rely on a consistent time reference to maintain quorum and cluster consistency. This event indicates that at least one of your mons is not sync'd correctly.  Review the cluster status with ceph -s. This will show which monitors are affected. Check the time sync status on each monitor host. ",
```

Comment:
"Monitors require closely-aligned time in order to maintain quorum and cluster consistency, but at least one of your mons is not synchronized closely.  Review cluster status with `ceph -s` and `ceph time-sync-status`.  This will show which monitors are affected.  Note that the latter reports offset relative to the current lead mon."